app/views/install/compose.phtml

```diff
@@ -351,6 +351,9 @@ services:
     restart: unless-stopped
     networks:
       - appwrite
+	  volumes:
+      - appwrite-functions:/storage/functions:rw
+      - appwrite-builds:/storage/builds:rw
```

Comment:
Incorrect indentation...and I'm surprised this isn't conflicting with https://github.com/appwrite/appwrite/pull/6133/files

Reply:
Weird, it shows okay on my end
<img width="970" alt="CleanShot 2023-09-06 at 10 13 12@2x" src="https://github.com/appwrite/appwrite/assets/19310830/92b80d35-7bc9-44da-a1f7-5e8db10e57b5">

Reply:
Rewrote it, good now. some space/tab issues